### PR TITLE
fix: Prompting for hostname for internal image registry(GITOPS-419)

### PR DESCRIPTION
**Pull request by iam-veeramalla**
> /kind bug

**What does this PR do / why we need it**:
Removes the Prompting for hostname for internal image registry. 

**Have you updated the necessary documentation?**
https://issues.redhat.com/browse/GITOPS-419

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/GITOPS-419

**How to test changes / Special notes to the reviewer**:
Run `kam bootstrap`

### CODE REVIEW

File: pkg/pipelines/imagerepo/imagerepo.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ import (
 
 // ValidateImageRepo validates the input image repo.  It determines if it is
 // for internal registry and prepend internal registry hostname if necessary.
-func ValidateImageRepo(imageRepo, registryURL string) (bool, string, error) {
+func ValidateImageRepo(imageRepo string) (bool, string, error) {
+	registryURL := "image-registry.openshift-image-registry.svc:5000"
```
Comment (bigkevmcd) [inline pkg/pipelines/imagerepo/imagerepo.go:22]: This could be moved to a top-level constant for visibility.

**Review by bigkevmcd — COMMENTED** [review]
Looks good, you'll need to run `make cmd-docs` as the failing check indicates.

**Comment by iam-veeramalla** [comment]
> Looks good, you'll need to run `make cmd-docs` as the failing check indicates.

I don't see any issue running that command. I will re-try .. pipelines are also failing pulling images. Will have a look

File: pkg/pipelines/imagerepo/imagerepo.go (hunk 1 above)
Comment (iam-veeramalla) [reply pkg/pipelines/imagerepo/imagerepo.go]: make the required changes

**Comment by iam-veeramalla** [comment]
@bigkevmcd Can you please approve this.

File: pkg/pipelines/imagerepo/imagerepo.go [hunk 2]
```diff
@@ -16,9 +16,11 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const registryURL = "image-registry.openshift-image-registry.svc:5000"
```
Comment (bigkevmcd) [inline pkg/pipelines/imagerepo/imagerepo.go:19]: I'd probably have named this `internalRegistryURL` but this is probably fine.

**Review by bigkevmcd — APPROVED** [review]

**Pull request merged**